dsbridge/DWKwebview.m

```diff
@@ -11,11 +11,13 @@
 
 @implementation DWKwebview
 {
-    void(^alertHandler)(void);
+    void (^alertHandler)(void);
     void (^confirmHandler)(BOOL);
     void (^promptHandler)(NSString *);
     int dialogType;
     UITextField *txtName;
+    int counter;
+    NSMutableDictionary *callbackDict;
```

Comment:
i don't know how to keep a callback dictionary, need some help.

`callbackDict` should have callbackId as its keys, and complete handlers as its values.

Reply:
That's the correct kind of object to store the dictionary of callbacks. 
Where would we need to set the values and retrieve them? 

The syntax to set would be [callbackDict setObject:callback forKey:calbackId];

Reply:
thanks! please help CR this new created commit: https://github.com/Tubitv/DSBridge-IOS/pull/1/commits/7adb440f06035a11b82006e87f1d645276868ca7.